JetUtilities/src/L2Creator.cc

```diff
@@ -43,36 +50,36 @@ L2Creator::L2Creator(CommandLine& cl) {
   mpv        = cl.getValue<bool>    ("mpv",            false);
   delphes    = cl.getValue<bool>    ("delphes",        false);
   maxFitIter = cl.getValue<int>     ("maxFitIter",        30);
+  cbPar      = cl.getValue<int>     ("cbPar", -1);
 
   if (!cl.partialCheck()) return;
   cl.print();
 }
 
 void L2Creator::resetForNextAlgorithm() {
   if(ji!=0)
-  	delete ji;
+    delete ji;
   if(odir!=0)
-  	delete odir;
+    delete odir;
   if(l3dir!=0)
-  	delete l3dir;
-	if(gl3rsp!=0)
+    delete l3dir;
+  if(gl3rsp!=0)
     delete gl3rsp;
-	if(fl3rsp!=0)
+  if(fl3rsp!=0)
     delete fl3rsp;
 
   hl_rsp.reset();
   hl_refpt.reset();
-	hl_jetpt.reset();
+  hl_jetpt.reset();
 
-	vabsrsp_eta.erase(vabsrsp_eta.begin(),vabsrsp_eta.end());
+  vabsrsp_eta.erase(vabsrsp_eta.begin(),vabsrsp_eta.end());
   vabscor_eta.erase(vabscor_eta.begin(),vabscor_eta.end());
   vrelcor_eta.erase(vrelcor_eta.begin(),vrelcor_eta.end());
 }
 
 //______________________________________________________________________________
 void L2Creator::openOutputFile() {
-  if(!outputDir.EndsWith("/")) outputDir+="/";
- 	ofile = TFile::Open(outputDir+output,"RECREATE");
+  ofile = TFile::Open(output,"RECREATE");
```

Comment:
Because I already parse all the path directly from the py file. It is more simple for me. I can add a flag to use that only for tau study?